scripts/update-cni-images.sh

```diff
@@ -30,4 +30,4 @@ fi
 echo "Applying amazon-vpc-cni-k8s/config/master/aws-k8s-cni.yaml manifest"
 kubectl apply -f $AWS_K8S_CNI_MANIFEST
 
-check_ds_rollout "aws-node" "kube-system" "4m"
+check_ds_rollout "aws-node" "kube-system" "10m"
```

Comment:
10m seems to be a lot. Instead we should consider setting this value based on the number of nodes or aws node pod instances in the cluster. For now it is fine but we should fine tune it in future.

Reply:
This doesn't wait 10 minutes by default, it waits up to 10 minutes before calling the operation a failure. This just ends up calling `kubectl rollout status ds/aws-node -n kube-system --timeout=10m`. So it is no slower than what is already there